src/App.css

```diff
@@ -1,7 +1,6 @@
 #root {
   max-width: 1280px;
```

Comment:
```suggestion
```
Remove this line, because otherwise we won't have a full navbar on wider screens (e.g. 1920px).